jni/src/knn_extension/faiss/utils/BitSet.cpp

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "BitSet.h"
+
+FixedBitSet::FixedBitSet(const int* intArray, const int length){
+    int maxValue = intArray[length - 1];
+    this->n = (maxValue >> 6) + 1; // div by 64
```

Comment:
What are we doing here?

Reply:
Calculating the size of required number of uint64 to store all int in bits.